lms/djangoapps/ccx/migrations/0004_seed_forum_roles_in_ccx_courses.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from ccx_keys.locator import CCXLocator
+from courseware.courses import get_course_by_id
+from django.db import migrations
+from django.http import Http404
+
+from django_comment_common.models import FORUM_ROLE_ADMINISTRATOR, FORUM_ROLE_MODERATOR, \
+    FORUM_ROLE_COMMUNITY_TA, FORUM_ROLE_STUDENT
+from django_comment_common.utils import STUDENT_ROLE_PERMISSIONS,  MODERATOR_ROLE_PERMISSIONS, \
+    ADMINISTRATOR_ROLE_PERMISSIONS
+
+
+def seed_forum_roles_for_existing_ccx(apps, schema_editor):
```

Comment:
@ormsbee I believe it is still necessary. #13196 grants the coach staff permissions on the CCX course, but we still need to create forum roles/permissions for existing CCX courses and grant relevant permissions to the coach.